core/src/com/unciv/models/ruleset/Ruleset.kt

```diff
@@ -510,4 +510,8 @@ class Ruleset {
     }
 
     fun getErrorList(tryFixUnknownUniques: Boolean = false) = RulesetValidator(this).getErrorList(tryFixUnknownUniques)
+
+    fun getResource(resourceName: String): TileResource {
```

Comment:
Unnecessary and crash-prone - with mods for can never be sure to get a resource from a name